client/js/entry-client-timeout.js

```diff
@@ -0,0 +1,3 @@
+// Assets
+import 'hydrogen-view-sdk/assets/theme-element-light.css';
+import '../css/styles.css';
```

Comment:
Adding an entrypoint just so we can pull in the necessary assets for the page and have it mapped in `manifest.json`